filter.go

```diff
@@ -0,0 +1,112 @@
+package reviewdog
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/haya14busa/reviewdog/diff"
+)
+
+type FilteredCheck struct {
+	*CheckResult
+	InDiff   bool
+	LnumDiff int
+}
+
+func FilterCheck(results []*CheckResult, diff []*diff.FileDiff, strip int, wd string) []*FilteredCheck {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function FilterCheck should have comment or be unexported